Logging to shared log directory: /tmp/test_logs//1333467113
only running test 15a
excepting tests: 15c
Starting client mft02: -o user_xattr,acl,flock mft02@tcp:/lustre /mnt/lustre2
Started clients mft02: 
mft02@tcp:/lustre on /mnt/lustre2 type lustre (rw,user_xattr,acl,flock)
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
mft02: Checking config lustre mounted on /mnt/lustre
Checking servers environments
Checking clients mft02 environments
Using TIMEOUT=20
disable quota as required
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
setting all flavor to null
already have total 8 null connections
...............

== replay-dual test 15a: timeout waiting for lost client during replay, 1 client completes == 18:32:04 (1333467124)
Filesystem           1K-blocks      Used Available Use% Mounted on
mft02@tcp:/lustre       374928     49528    305272  14% /mnt/lustre
total: 25 creates in 0.03 seconds: 854.09 creates/second
total: 1 creates in 0.03 seconds: 31.88 creates/second
Failing mds1 on node mft02
Stopping /mnt/mds1 (opts:) on mft02
affected facets: mds1
Failover mds1 to mft02
18:32:18 (1333467138) waiting for mft02 network 900 secs ...
18:32:18 (1333467138) network interface is UP
Starting mds1: -o loop,user_xattr,acl  /tmp/lustre-mdt1 /mnt/mds1
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
Started lustre-MDT0000
 - unlinked 0 (time 1333467230 ; total 0 ; last 0)
total: 25 unlinks in 0 seconds: inf unlinks/second
Starting client: mft02: -o user_xattr,acl,flock mft02@tcp:/lustre /mnt/lustre2
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
Resetting fail_loc on all nodes...done.
PASS 15a (106s)
........== replay-dual replay-dual.sh test complete, duration 117 sec == 18:33:50 (1333467230)
Stopping clients: mft02 /mnt/lustre2 (opts:)
Stopping client mft02 /mnt/lustre2 opts:
